crates/release_plz_core/src/release_pr.rs

```diff
@@ -169,6 +169,8 @@ fn reset_branch(pr: &GitPr, commits_number: usize, repository: &Repo) -> anyhow:
     let head = format!("HEAD~{commits_number}");
     repository.git(&["reset", "--hard", &head])?;
 
+    repository.git(&["fetch"])?;
```

Comment:
Yes exactly. Since fetch does not change anything to the staged / working set files it could also be executed before the `git reset` call. But I guess it's not important for now.